### DISCUSSION TITLE
Css Framework Addition: Ant Design

### DISCUSSION THREAD

**Discussion by daxaxelrod** [abishekmuthian/startuptoolchain#4] [Ideas]
Ant has a fantastic CSS lib that feels clean & modern. Plus they also have an excellent React component library. Would love to see it added to the Css Framework framework section.

[Website](https://ant.design/)

**Comment by abishekmuthian** [comment]
Very interesting, Thanks for sharing.

Looks like it's being [used predominantly by apps in China](https://github.com/ant-design/ant-design/issues/477) wouldn't be useful under [Design Language](https://startuptoolchain.com/#design-language) so those who want to build solutions for China could make use of it.
